examples/js/loaders/GLTFLoader.js

```diff
@@ -27,6 +27,27 @@ THREE.GLTFLoader = ( function () {
 
 			var path = this.path !== undefined ? this.path : THREE.LoaderUtils.extractUrlBase( url );
 
+			var itemGroup = '[group: ' + url + ']';
```

Comment:
Isn't just `url` good here? I suppose user wants `url` (*.gltf/.glb) as an argument for `onStart()` of `LoadingManager`.

Reply:
FileLoader will call itemStart / itemEnd with `url` when loading it, so I don't want to conflict with the real URL — I think we'd end up calling `manager.onLoad()` early then.

Reply:
Can Loader or LoadingManager fail if conflict?

I haven't used `LoadingManager` so much, then not sure if non-valid url is ok as an argument of `onStart()` for current and potential users. I suppose user expects real(or valid) url from the `LoadingManager` document https://threejs.org/docs/#api/loaders/managers/LoadingManager.onStart Any thoughts? /cc @Mugen87 @mrdoob 

If it's ok, I think we should add note to `GLTFLoader` doc.

Reply:
The url is passed to `onStart()`, `onProgress()` and `onError()` as a callback parameter. I don't know how this value is normally used. I guess if anything the value is printed to the console for monitoring purposes.

> so I don't want to conflict with the real URL 

I don't think this can happen since `LoadingManager` just counts the request and does not use the URL as an identifier. The same URL should actually work.

Reply:
The conflict means you get `onStart()` called twice with the same URL. It would still work, I guess, although relying on it to track two items with the same name separately feels a little unstable.

I'd rather not have users expect this URL or rely on it, which documentation might encourage... It's really a hack to ensure onLoad is called at the right time, and if there is a better approach we can change this.

Reply:
^Oops, didn't see your reply before sending that @Mugen87, but yes agreed.

Reply:
> The conflict means you get onStart() called twice with the same URL.

If I understand correctly, `onStart()` with the root url (*.gltf/glb) is called only once. Because `isLoading` is already true when the second `.itemStart( url )` is called by `FileLoader` https://github.com/mrdoob/three.js/blob/dev/src/loaders/LoadingManager.js#L19-L35

Reply:
Oope, I've missed too.

> I don't think this can happen since LoadingManager just counts the requests and does not use the URL as an identifier. The same URL should actually work.

Reply:
Oh you're right! Since this is the first itemStart call and onStart() will fire, let's just use the same URL, then?

Reply:
Yeah, I prefer that if there's no problem.

Reply:
Plus, I'd like to add a note here (or to document) that we return incorrect `itemsLoaded/Total` as #14256 mentioned.